setup.py

```diff
@@ -257,6 +257,7 @@
 
 
 def main():
+    print('Hello there')
```

Comment:
no, you should use double quotes., and print to stderr with the `file=` argument.